modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -751,6 +751,23 @@ public void testPutObjectDefinition() throws Exception {
 						localized = true;
 						name = StringUtil.randomId();
 					}
+				},
+				new ObjectField() {
+					{
+						businessType = BusinessType.TEXT;
+						DBType = ObjectField.DBType.create("String");
+						label = Collections.singletonMap("en_US", "Column");
+						localized = false;
+						name = StringUtil.randomId();
+					}
+				},
+				new ObjectField() {
+					{
+						businessType = BusinessType.TEXT;
+						DBType = ObjectField.DBType.create("String");
+						label = Collections.singletonMap("en_US", "Column");
+						name = StringUtil.randomId();
+					}
```

Comment:
I believe this test could be more valuable.

Can we add a case where the field type does not support localization? Specifically, testing the scenario where localized = false and another where the property is not passed.